dags/iop/parser.py

```diff
@@ -28,18 +29,22 @@ def __init__(self) -> None:
                 destination="journal_doctype",
                 extraction_function=self._get_journal_doctype,
             ),
+            CustomExtractor(
+                destination="related_article_doi",
+                extraction_function=self._get_related_article_doi,
+            ),
         ]
         super().__init__(extractors)
 
     def _get_dois(self, article: ET.Element):
         node = article.find("front/article-meta/article-id/[@pub-id-type='doi']")
         if node is None:
-            return None
+            return
         dois = node.text
         if dois:
             self.logger.msg("Parsing dois for article", dois=dois)
             self.dois = dois
-            return [dois]
+            return dois
```

Comment:
why you changed it from a list? If you do so you have to also rename it, `doi`

Reply:
https://github.com/cern-sis/workflows/pull/67#discussion_r1045796997